### PR TITLE
Improve navigation

### PR DESCRIPTION
**Pull request by nekath**
### Description
- Move assets to separate controller
- Implement logout link
- Add homepage with CTA login button
- Disable robots

### CODE REVIEW

File: assets/static/robots.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 #
 # To ban all spiders from the entire site uncomment the next two lines:
-# User-agent: *
-# Disallow: /
+User-agent: *
+Disallow: /
```
Comment (kamil89) [inline assets/static/robots.txt]: +1

File: lib/sejfguru_web/controllers/asset_controller.ex [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    |> assign(:assets, fetch_assets(page))
+    |> render("index.html")
+  end
+
+  def index(conn, _params) do
+    conn
+    |> assign(:page, 1)
+    |> assign(:assets, fetch_assets(1))
+    |> render("index.html")
+  end
```
Comment (kamil89) [inline lib/sejfguru_web/controllers/asset_controller.ex]: I'd suggest extracting `page` fetching and having single `index` action.
```elixir
page = fetch_page(params)
def fetch_page(%{"page" => page}), do: page
def fetch_page(_params), do: 1
```

File: lib/sejfguru_web/templates/layout/navbar.html.eex [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<%= if assigns[:current_user] do %>
```
Comment (kamil89) [inline lib/sejfguru_web/templates/layout/navbar.html.eex]: Whole template is inside `if`. Maybe move `if` into place where template is called to render.

File: lib/sejfguru_web/templates/layout/navbar.html.eex [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        <div class="navbar-header">
+          <a class="navbar-brand" href="/">Sejfguru</a>
+        </div>
+        <ul class="nav navbar-nav">
+          <li class="active"><a href="<%= asset_path(@conn, :index) %>">Assets</a></li>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li class="dropdown">
+            <a href="" class="dropdown-toggle" id="dropdownNavbar"
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"
```
Comment (kamil89) [inline lib/sejfguru_web/templates/layout/navbar.html.eex]: I'd align `data-toggle` as `href` is.

Comment (nekath) [reply lib/sejfguru_web/templates/layout/navbar.html.eex]: We need some linter for stuff like this 😞

**Pull request merged**